mappings/net/minecraft/entity/ItemDirigible.mapping

```diff
@@ -0,0 +1,16 @@
+CLASS net/minecraft/class_4981 net/minecraft/entity/ItemDirigible
+	METHOD method_26313 travel (Lnet/minecraft/class_1308;Lnet/minecraft/class_4980;Lnet/minecraft/class_243;)Z
+		ARG 1 entity
+		ARG 2 saddledEntity
+		ARG 3 movementInput
+	METHOD method_26314 interactMob (Lnet/minecraft/class_1308;Lnet/minecraft/class_1657;Lnet/minecraft/class_1268;Z)Z
+		ARG 1 entity
+		ARG 2 player
+		ARG 3 hand
+		ARG 4 childAllowed
```

Comment:
ah sorry, should prob be `childRideable`
```suggestion
		ARG 4 childRideable
```
since this is not just riding logic but also name tag logic